ragaai_catalyst/guard_executor.py

```
@@ -2,26 +2,44 @@
+from google import genai
+from google.genai.types import GenerateContentConfig
+from typing import Optional, List, Dict, Any
 import logging
 logger = logging.getLogger('LiteLLM')
 logger.setLevel(logging.ERROR)
 
 class GuardExecutor:
 
-    def __init__(self,id,guard_manager,field_map={}):
-        self.deployment_id = id
+    def __init__(self,guard_manager,input_deployment_id = None,output_deployment_id=None,field_map={}):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)